tox.ini

```diff
@@ -72,6 +69,36 @@ basepython = python3.8
 [testenv:py3.9]
 basepython = python3.9
 
+[plugins]
+commands =
+    python -m pip install --no-deps file://{toxinidir}/plugins/aea-crypto-ethereum
+    python -m pip install --no-deps file://{toxinidir}/plugins/aea-crypto-cosmos
+    python -m pip install --no-deps file://{toxinidir}/plugins/aea-crypto-fetchai
+    pytest -rfE plugins/aea-crypto-fetchai/tests --cov-report=html --cov-report=xml --cov-report=term --cov-report=term-missing --cov=aea_crypto_fetchai --cov-config=.coveragerc --suppress-no-test-exit-code {posargs}
+    pytest -rfE plugins/aea-crypto-ethereum/tests --cov-report=html --cov-report=xml --cov-report=term --cov-report=term-missing --cov=aea_crypto_ethereum --cov-config=.coveragerc --suppress-no-test-exit-code {posargs}
+    pytest -rfE plugins/aea-crypto-cosmos/tests --cov-report=html --cov-report=xml --cov-report=term --cov-report=term-missing --cov=aea_crypto_cosmos --cov-config=.coveragerc --suppress-no-test-exit-code {posargs}
+
+[testenv:plugins-py3.6]
+basepython = python3.6
+commands = {[plugins]commands}
+
+[testenv:plugins-py3.7]
+basepython = python3.7
+commands = {[plugins]commands}
+
+[testenv:plugins-py3.7-cov]
+basepython = python3.7
+usedevelop = True
+commands = {[plugins]commands}
+
+[testenv:plugins-py3.8]
+basepython = python3.8
+commands = {[plugins]commands}
+
+[testenv:plugins-py3.9]
+basepython = python3.9
+commands = {[plugins]commands}
+
```

Comment:
I feel this might still be improved, as the "main testenv" above, still good enough.